chrome-extension/test/configure.js

```diff
@@ -1,4 +1,5 @@
 'use strict';
+process.exit();
```

Comment:
I asked because https://github.com/haystack/tipsy/pull/91/commits/4989d57b71641725e506f1095b183e7effb8307c (a later commit) appears to be trying to fix the chrome test. Does that fix not work?